docs/relying-parties/reference/metrics-for-relying-parties.md

```diff
@@ -4,7 +4,29 @@ title: Metrics for Relying Parties
 sidebar_label: Metrics
 ---
 
-Last updated: `January 26th, 2022`
+Last updated: `March 15th, 2023`
+
+## Subscriptions Services Data Sources
+Subscription Platform provides product metrics by obtaining subscriptions data from the following sources: Stripe, mobile app store providers, and FxA logs.
+
+### Stripe
+The bulk of Subscriptions Services data comes from Stripe and is synced to BigQuery using FiveTran and its Stripe connector. This data reflects the current subscription state of all active subscriptions paid for via Stripe or PayPal. It includes information, such as the customer's start and end dates of a subscription, etc.
+
+Other information includes customers' postal code, state, and country for a subscription, which helps resolve issues for customers across a particular dimension (e.g., country, payment provider, etc.). As the data intentionally excludes any personal identifiable information of customers (e.g., name, street address, phone number, email address), it makes it challenging to use this data to help on an individual customer level. Other uses for this data is for retrieving time counts (e.g., total number of active subscriptions).
```

Comment:
Help: the [ticket](https://mozilla-hub.atlassian.net/browse/FXA-6943) states that Stripe data is "useful for all time counts of things like total number of (active) subscriptions". What does "time counts" mean, what else does that include?

Reply:
I think that means _all-time_ counts (i.e. total counts).